### PR TITLE
Rescu readme revamp

### PR DESCRIPTION
**Pull request by radumas**
## What this pull request accomplishes:

- Incorporates the data stewardship questionnaire into the RESCU README
- 

## Issue(s) this solves:

- #565 

## What, in particular, needs to reviewed:

- 

## What needs to be done by a sysadmin after this PR is merged

Nothing

### CODE REVIEW

**Comment by radumas** [comment]
OH NO I deleted the image

File: volumes/rescu/README.md [hunk 1]
```diff
@@ -1,11 +1,390 @@
-# RESCU - Vehicle Detector Stations
+
+# RESCU Vehicle Detector Stations <!-- omit in toc -->
 
 Road Emergency Services Communication Unit (RESCU) data tracks traffic volume on expressways using loop detectors. 
 Some more information can be found on r [here](https://en.wikipedia.org/wiki/Road_Emergency_Services_Communications_Unit).
```
Comment (gabrielwol) [inline volumes/rescu/README.md:5]: Typo? found on **r**?

File: volumes/rescu/README.md [hunk 2]
```diff
@@ -1,11 +1,391 @@
...
+
+## Schema
+
+- Data is stored in the `rescu` schema.
+- The main tables are listed below. Please note that `rescu.volumes_15min` and `rescu.detector_inventory` are the tables that should be used for querying.
+
+### Data Dictionary:
+
+`rescu.detector_inventory` table: 
+This table contains list and details of RESCU detectors in the City. Please note this table needs to be updated.
```
Comment (gabrielwol) [inline volumes/rescu/README.md:60]: "this table needs to be updated" - please expand

**Review by gabrielwol — APPROVED** [review]
I fixed up the data dictionaries and addressed some other minor comments I had. Approved to merge.

**Comment by radumas** [comment]
Merging this, missing data flow diagram can be dealt with in the new VDS pipeline.

**Pull request closed without merging**